src/views/Dissolution/DissolutionReviewConfirm.vue

```diff
@@ -254,7 +254,6 @@
         <CourtOrderPoa
           class="py-8 px-6"
           :class="{ 'invalid-section': isCourtOrderInvalid }"
-          :autoValidation="getValidateSteps"
```

Comment:
Did you test that dissolution filings work acceptably (same as IAs)?

Reply:
No, good call. I haven't created a dissolution filing yet so I might need some help getting it setup, but I'll give it a try on my own first.